src/Core/tests/DeviceTests/Platforms/Android/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
-	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="30" />
```

Comment:
This project was giving a build warning, because .NET 6 is API 31. We can just remove this so you don't have to keep updating it.